ci/run-bridge.sh

```diff
@@ -9,7 +9,7 @@ NODE_NAME="bridge-$NODE_ID"
 # a private local network
 P2P_NETWORK="private"
 # a bridge node configuration directory
-CONFIG_DIR="$CELESTIA_HOME/.celestia-bridge-$P2P_NETWORK"
+CONFIG_DIR="$HOME/.celestia-bridge-$P2P_NETWORK"
```

Comment:
why this change? :thinking:

Reply:
`CELESTIA_HOME` behavior [changed](https://github.com/celestiaorg/celestia-node/blob/main/nodebuilder/store.go#L189), it must now point to the exact directory where node will have it's config. In docker image we were setting it to just `HOME`. If you leave it unset, node will still set up using the previous path schema tho :thinking: not sure about reasoning, but there was something in release notes

Reply:
I see, thank you :raised_hands: